src/backend/distributed/shared_library_init.c

```diff
@@ -358,19 +358,6 @@ RegisterCitusConfigVariables(void)
 		0,
 		NULL, NULL, NULL);
 
-	DefineCustomBoolVariable(
```

Comment:
well, we shouldn't remove the GUC, no? Should we make `ExpireCachedShards ` a static variable and add a comment here that the GUC is not used anymore?